README.md

```diff
@@ -1,6 +1,82 @@
+<img width="128" height="128" alt="superhero-dev" src="https://github.com/user-attachments/assets/c716fb81-27e3-4acf-a873-e6425a39e136" />
 
-<img width="256" height="256" alt="superhero-dev" src="https://github.com/user-attachments/assets/c716fb81-27e3-4acf-a873-e6425a39e136" />
+# AI-Enablement Prompts
 
-# AI Enablement Prompts
+**AI-Enablement Prompts** is a collection of advanced AI prompt chains created by [Bitovi](https://www.bitovi.com), a consultancy on the cutting edge of AI-augmented software development. These prompts are designed to help AI agents like GitHub Copilot move beyond simple autocomplete and become valuable, context-aware teammates.
 
-This is a collection of useful prompts for AI-enabled software development.
+Each prompt or prompt chain is built to guide the AI through specific engineering workflows — analyzing a codebase, generating documentation, implementing features, and more — using clearly defined, repeatable steps that reflect how real engineers work.
+
+Our goal is to bridge the gap between general-purpose AI models and project-specific knowledge, enabling faster, more accurate, and more scalable development.
+
+## Repository Structure
+
+Prompts in this repo are organized into **high-level categories** based on what they help the AI accomplish:
+
+- **`/understanding-code`** – Prompt chains that help AI understand and document existing codebases
+- **`/writing-code`** – Prompt chains that assist with implementing new functionality based on external input (e.g. Jira tickets)
+
+Each subfolder contains one or more prompts or prompt chains, each with its own `README.md` explaining how to use it.
+
+
+## What's Inside
+
+Here are a few currently available prompt chains:
+
+### 1. Developer Docs Generator  
+**Path:** `understanding-code/developer-docs`
+
+Generates a `copilot-instruction.md` file that helps AI tools operate effectively within your codebase. Ideal for onboarding AI tools (or humans) with deep, structured context.
```

Comment:
I think there's a missunderstanding here.  This should be `understanding-code/instruction-generation`.  "Developer Docs" was what `folder-overview` would eventually do.  Creating instructions is something else.